lib/LMSManagers/LMSCustomerManager.php

```diff
@@ -657,6 +657,7 @@ public function customerAdd($customeradd)
             'extid'          => $customeradd['extid'],
             'name'           => $customeradd['name'],
             'lastname'       => $customeradd['lastname'],
+            'altname'        => empty($customeradd['altname']) ? '' : $customeradd['altname'],
```

Comment:
Dlaczego nie bardziej elegancki `null` w przypadku, gdy brak nazwy alternatywnej?

Reply:
@chilek zadawałem to pytanie kilkukrotnie:

Jak lepiej i dlaczego:
`empty($customeradd['altname']) ? null : $customeradd['altname']`
`empty($customeradd['altname']) ?: $customeradd['altname']`
?